#!/usr/bin/env python3
PACKAGE = "asv_sim"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("current_speed",        double_t, 0, "Speed of current, in m/s",                  1.0, 0.0, 5.0)
gen.add("current_direction",    double_t, 0, "Direction current is flowing, in degrees", 90.0, 0.0, 360.0)

exit(gen.generate(PACKAGE, "asv_sim", "environment"))
